packages/backend/src/guards/firebare.auth.ts

```diff
@@ -0,0 +1,35 @@
+import { PassportStrategy } from '@nestjs/passport';
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { Strategy, ExtractJwt } from 'passport-firebase-jwt';
+import { initializeApp, cert } from 'firebase-admin/app';
+import { getAuth } from 'firebase-admin/auth';
+
+@Injectable()
+export class FirebaseAuthStrategy extends PassportStrategy(
+  Strategy,
+  'firebase-auth',
+) {
+  constructor() {
+    super({
+      jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+    });
+    initializeApp({
+      credential: cert(require('../../../../keys/firebase.json')),
```

Comment:
This path could possibly be moved to an env, but I'm hesitant since it complicates the setup process.

Reply:
yeah I agree with you. I think we really need to make the developer experience with the env much better